docs/src/reference/the-traversal.asciidoc

```diff
@@ -1916,6 +1916,35 @@ g.V(1).both().both().simplePath()
 g.V(1).both().both().simplePath().path()
 ----
 
+[[skip-step]]
+Skip Step
+~~~~~~~~~
+
+The `skip()`-step is analogous to <<range-step,`range()`-step>> save that the higher end range is set to -1.
+
+[gremlin-groovy,modern]
+----
+g.V().skip(2)
+g.V().range(2, -1)
+g.V().skip(2).toString()
```

Comment:
We should not advertise `toString()` as other GLV's don't show compilation information and in fact, down the line, we will most likely change what `toString()` shows in Gremlin-Java/Groovy.

I would add an example that uses `order().by(...).skip(x)` so that people know it skips the first `x` results. For instance, show without `skip(x)`, then show with `skip(x)` after an `order()`.